src/chapters/style/documentation.typ

```diff
@@ -0,0 +1,247 @@
+#import "/src/util.typ": *
+#import mantys: *
+
+= Documentation <sec:style:docs>
+#wbox[
+  This section documents the syntax of documentation commens, see @sec:api:docs for the contents and purpose of documentation comments.
+]
+
+Documentation is placed on special comments using three forward slashes `///` and an optional space.
+These are called doc comments.
+While the leading space is optional, it is encouraged, as it makes the documentation easier to read.
+Doc comments may not be interrupted by empty lines, markup, or regular comments, i.e. a blokc of documenation must be one continuous sequence of lines starting with three forward slashes `///`.
+
+#do-dont[
+  ```typst
+  /// A1
+  /// A2
+  /// A3
+
+  // discuraged, but valid
+  ///B1
+  ///B2
+  ///B3
+  ```
+][
+  ```typst
+  /// A1
+  //
+  /// B1
+
+  /// A1
+
+  /// B1
+
+  /// C1
+  Hello World
+  /// D1
+
+  /**
+   * This is a regular comment, not a doc comment.
+   */
+  #let item = { ... }
+
+  // This is a regular comment, not a doc comment.
+  #let item = { ... }
+  ```
+]
+
+There are two kinds of documentation comments, inner and outer doc comments.
```

Comment:
"Outer" is okay but the name "inner comment" does not really make sense. It is sort of even more outer than the doc comments for declarations because it refers to the entire module. 

I get it that they are declared with_in_ the module but it is very confusing